### PR TITLE
Update front matter, acknowledgements and criticality for AAD

### PR DESCRIPTION
**Pull request by ahuynhMITRE**
## 🗣 Description ##

Updating the front-matter and acknowledgements to the AAD SCB to match language approved by the CISA Front office. Also added the html formatting for criticality to support new field in report. 

## 💭 Motivation and context ##

Changes are required to ensure consistency across SCBs for the approved front-matter and acknowledgements. Criticality HTML is added to support new report field and supporting automation.  

#471 
#341 

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

File: baselines/aad.md [hunk 1]
```diff
@@ -2,31 +2,21 @@
...
-International license. Source documents are linked throughout this
-document. The United States government has adapted selections of these
-documents to develop innovative and scalable configuration standards to
-strengthen the security of widely used cloud-based software services.
+The Secure Cloud Business Applications (SCuBA) project run by the Cybersecurity and Infrastructure Security Agency (CISA) provides guidance and capabilities to secure federal civilian executive branch (FCEB) agencies’ cloud business application environments and protect federal information that is created, accessed, shared, and stored in those environments. 
 
-## Assumptions
+The CISA SCuBA SCBs for M365 help secure federal information assets stored within M365 cloud business application environments through consistent, effective, and manageable security configurations. CISA created baselines tailored to the federal government’s threats and risk tolerance with the knowledge that every organization has different threat models and risk tolerance. Non-governmental organizations may also find value in applying these baselines to reduce risks.
```
Comment (crutchfield) [inline baselines/aad.md:7]:
```suggestion
The CISA SCuBA Security Configuration Baselines (SCB) for M365 help secure federal information assets stored within M365 cloud business application environments through consistent, effective, and manageable security configurations. CISA created baselines tailored to the federal government’s threats and risk tolerance with the knowledge that every organization has different threat models and risk tolerance. Non-governmental organizations may also find value in applying these baselines to reduce risks.
```

File: baselines/aad.md [hunk 2]
```diff
@@ -104,17 +91,22 @@ This section provides policies that help reduce security risks related to user a
 #### MS.AAD.2.1v1
 Users detected as high risk SHALL be blocked.
 
+<!--Policy: MS.AAD.2.1v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/aad.md:94]: See comments in other PR review.  Criticality needs to include additional info. 
```suggestion
<!--Policy: MS.AAD.2.1v1; Criticality: SHALL/NOT-IMPLEMENTED -->
```

File: baselines/aad.md (hunk 1 above)
Comment (ahuynhMITRE) [reply baselines/aad.md:7]: added to first paragraph instead to match example.

File: baselines/aad.md [hunk 3]
```diff
@@ -1,32 +1,22 @@
 # CISA M365 Security Configuration Baseline for Azure Active Directory
 
-Azure Active Directory (AAD) is a cloud-based identity and access control service that provides security and functional capabilities to Microsoft 365. This security baseline provides policies to help secure AAD. 
+Azure Active Directory (AAD) is a cloud-based identity and access control service that provides security and functional capabilities to Microsoft 365. his Secure Configuration Baseline (SCB) provides specific policies to help secure AAD.
```
Comment (crutchfield) [inline baselines/aad.md:3]:
```suggestion
Azure Active Directory (AAD) is a cloud-based identity and access control service that provides security and functional capabilities to Microsoft 365. This Secure Configuration Baseline (SCB) provides specific policies to help secure AAD. 
```

File: baselines/aad.md [hunk 4]
```diff
@@ -104,17 +91,22 @@ This section provides policies that help reduce security risks related to user a
...
 
+<!--Policy: MS.AAD.2.1v1; Criticality: SHALL -->
 - _Rationale:_ By blocking users determined as high risk, compromised accounts can be prevented from accessing the tenant.
 - _Last modified:_ June 2023
 - _Note:_ Users who are determined to be high risk by AAD Identity Protection can be blocked from accessing the system via an AAD Conditional Access policy. A high risk user will be blocked until an administrator remediates their account.
 
 #### MS.AAD.2.2v1
 A notification SHOULD be sent to the administrator when high-risk users are detected.
+
+<!--Policy: MS.AAD.2.2v1; Criticality: SHOULD -->
```
Comment (crutchfield) [inline baselines/aad.md:102]:
```suggestion
<!--Policy: MS.AAD.2.2v1; Criticality: SHOULD/NOT-IMPLEMENTED -->
```

File: baselines/aad.md [hunk 5]
```diff
@@ -188,52 +180,60 @@ The methods **AAD Certificate-Based Authentication (CBA)**, **FIDO2 Security Key
...
 
+<!--Policy: MS.AAD.3.2v1; Criticality: SHALL -->
 - _Rationale:_ This is a backup security policy to help protect the tenant in the event that phishing-resistant MFA has not been enforced yet. This policy requires that MFA is enforced and thus reduces the risks of single form authentication.
 - _Last modified:_ June 2023
 - _Note:_ If a conditional access policy has been created that enforces phishing-resistant MFA, then this policy is not necessary. This policy does not dictate the specific MFA method.
 
 #### MS.AAD.3.3v1
 If phishing-resistant MFA has not been enforced yet and Microsoft Authenticator is enabled, it SHALL be configured to show login context information.
 
+<!--Policy: MS.AAD.3.3v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/aad.md:198]: Rego has Should/Not-Implemented.  I see control reads as SHALL so I assume it should be SHALL/NOT-IMPLEMENTED.  Need issue for Rego?

File: baselines/aad.md [hunk 6]
```diff
@@ -188,52 +180,60 @@ The methods **AAD Certificate-Based Authentication (CBA)**, **FIDO2 Security Key
...
 
+<!--Policy: MS.AAD.3.4v1; Criticality: SHALL -->
 - _Rationale:_ By configuring the Manage Migration feature to Migration Complete, we ensure that the tenant has disabled the legacy authentication methods screen. The MFA and SSPR authentication methods are both managed from a central admin page thereby reducing administrative complexity and
 reducing the chances of security misconfigurations.
 - _Last modified:_ June 2023
 
 #### MS.AAD.3.5v1
 The authentication methods SMS, Voice Call, and Email OTP SHALL be disabled.
 
+<!--Policy: MS.AAD.3.5v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/aad.md:213]:
```suggestion
<!--Policy: MS.AAD.3.5v1; Criticality: SHALL/NOT-IMPLEMENTED -->
```

File: baselines/aad.md [hunk 7]
```diff
@@ -344,6 +344,7 @@ This section provides policies that help reduce security risks related to  the l
 #### MS.AAD.4.1v1
 Security logs SHALL be sent to the agency's Security Operations Center for monitoring.
 
+<!--Policy: MS.AAD.4.1v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/aad.md:347]:
```suggestion
<!--Policy: MS.AAD.4.1v1; Criticality: SHALL/NOT-IMPLEMENTED -->
```

**Review by crutchfield — CHANGES_REQUESTED** [review]
Looks good.  Couple minor changes need to be resolved.

**Comment by ahuynhMITRE** [comment]
> 

Committed your request, thanks!

**Review by nanda-katikaneni — APPROVED** [review]
Looks good.

File: baselines/aad.md [hunk 8]
```diff
@@ -391,24 +392,28 @@ This section provides policies that help reduce security risks related to  non-p
...
 An admin consent workflow SHALL be configured for applications.
 
+<!--Policy: MS.AAD.5.3v1; Criticality: SHALL -->
 - _Rationale:_ Configuring an admin consent workflow helps support the risk reduction of the previous policy by setting up a process for users to securely request access to applications necessary for business purposes. Administrators get the opportunity to review the permissions requested by new applications and approve or deny access based on a risk assessment.
 - _Last modified:_ June 2023
 
 #### MS.AAD.5.4v1
 Group owners SHALL NOT be allowed to consent to applications.
 
+<!--Policy: MS.AAD.5.3v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/aad.md:416]:
```suggestion
<!--Policy: MS.AAD.5.4v1; Criticality: SHALL -->
```

**Review by crutchfield — CHANGES_REQUESTED** [review]
1 minor change.

**Comment by ahuynhMITRE** [comment]
> 1 minor change.

committed thanks crutch!

**Review by ahuynhMITRE — COMMENTED** [review]
committed

**Review by crutchfield — APPROVED** [review]
Looks good.

**Comment by ahuynhMITRE** [comment]
@nanda-katikaneni good to merge!

**Comment by nanda-katikaneni** [comment]
@ahuynhMITRE there is a conflicted file: baselines/aad.md - can you please check and resolve.

**Comment by ahuynhMITRE** [comment]
> @ahuynhMITRE there is a conflicted file: baselines/aad.md - can you please check and resolve.

good to go now!

**Pull request merged**